pybaseball/teamid_lookup.py

```diff
@@ -33,6 +33,26 @@ def team_ids(season: Optional[int] = None, league: str = 'ALL') -> pd.DataFrame:
     return fg_team_data
 
 
+def mlb_com_team_id(teamName = None):
+    
+    '''
+        Provides team ids use within mlb.com urls
+        
+        if no team is given all team ids are returned in a dataframe; else a single team id is returned as a string
+    '''
+    
```

Comment:
Can you sanitize the input? I'd like to see `bluejays`, `blue_jays`, `blue jays`, or `Blue Jays` all work here. I think this should just be a string lower and remove spaces and underscores.